create_magic.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ARCH=x86_64
+TYPE=qcow2
+DIR=~/build
```

Comment:
$HOME/build assumes the directory has `chmod go+rx $HOME`  -- without this,  QEMU process cannot access the disk image. Also, the default location for images is: /var/lib/libvirt/images